subtitles/zh-CN/31_navigating-the-model-hub.srt

```diff
@@ -20,42 +20,42 @@ This is the huggingface.co landing page.
 
 5
 00:00:13,260 --> 00:00:16,020
-要访问模型中心，请单击模型选项卡
-To access the model hub, click on the models tab
+要访问 Model Hub，请在右上角
+To access the Model Hub, click on the models tab
 
 6
 00:00:16,020 --> 00:00:17,463
-在右上角。
+单击 model 选项卡。

```

Comment:
It is recommended to add an s here, so that it can be consistent with the label on the web page

Reply:
Sure